### DISCUSSION TITLE
Problem with Symfony 6

### DISCUSSION THREAD

**Discussion by mdeherder** [symfony/symfony#44047] [Q&A] [answered]
I try to install a symfony test application in version 6 following the scenario of the book 'The fast track
I'm quickly blocked, as soon as I install the 'dump' bundle
Scenario :
`composer create-project symfony/skeleton app6full -s dev`
`composer require profiler --dev`
`composer require logger`
So far, everything is ok
But after
`composer require debug --dev`
everything is broken
![Image 106](https://user-images.githubusercontent.com/29018936/141684806-78fed281-e359-4f89-8a21-bc70f961da60.png)
n

**Comment by chapterjason** [comment]
Hey,

I tried to reproduce, but it doesn't end up with an error. Did you changed anything else?

**Comment by mdeherder** [comment]
The BETA version has changed. My problem was related to version 6.0.0-BETA1

**Comment by mdeherder** [comment]
A second problem already present in BETA1 is not solved by BETA2.
After : 

> symfony console make:auth
> - Login form authenticator
> - AppAuthenticator
> - SecurityController
> - yes

I need to replace

`public function authenticate(Request $request): PassportInterface`

by

`public function authenticate(Request $request): Passport`

in the file `src/Security/AppAuthenticator.php` created by the command

**Comment by fabpot** [comment, marked as answer]
@mdeherder Can you report this issue to symfony/maker-bundle?

**Reply by chapterjason** [reply]
If I remember correctly, the `PassportInterface` got removed?

**Comment by mdeherder** [comment]
See issue symfony/maker-bundle#1011 on symfony/maker-bundle
